CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+### Version 0.12.1.beta
```

Comment:
:+1: on updating changelog header to `master` instead. Will revert actually bumping the version.